04week/loop.js

```diff
@@ -0,0 +1,72 @@
+'use strict'
+
+// Use a for loop to console.log each item in the array carsInReverse.
+
+const carsInReverse = ['Toyota', 'Mazda', 'Ford', 'Honda']
+
+carsInReverse.forEach(function(item){
+  console.log(item)
+});
+
+const actualForLoop = () =>{
+  for (i = 0; i < carsInReverse.length; i++)
+    console.log(carsInReverse[i])
+}
+
+// for...in loop
+// Create an object (an array with keys and values) called persons with the following data:
+// firstName: "Jane"
+// lastName: "Doe"
+// birthDate: "Jan 5, 1925"
+// gender: "female"
+  
+const myObject = {
+  firstName: "Jane",
+  lastName: "Doe",
+  birthDate: "Jan 5, 1925",
+  gender: "female"
+}
+
+// Use a for...in loop to console.log each key.
+
+for (key in myObject){
+  console.log(key)
+}
+
+// Then use a for...in loop and if state to console.log the value associated with the key birthDate.
+
+for (let key in myObject){
+  console.log(myObject[key])
+} // I didn't fully understand the directions on this question, are we supposed to use an if statement?
+  
+
+// while loop
+// Use a for loop to console.log the numbers 1 to 1000.
+
+for (i = 1; i < 1001; i++){
```

Comment:
' i  ' needs to be defined in the loop those wise it is being defined by the global variable. This could be something that you may have intend to do but for this assignment i know that this is not intended.